researchstack-sdk/build.gradle

```diff
@@ -55,5 +55,7 @@ dependencies {
   testCompile 'junit:junit:4.12'
   testCompile 'org.robolectric:robolectric:3.0'
   testCompile 'org.mockito:mockito-core:1.10.19'
+  testCompile 'org.powermock:powermock-api-mockito:1.6.6'
+  testCompile 'org.powermock:powermock-module-junit4:1.6.6'
```

Comment:
Hmm.. didn't work for me after a clean/build. Are test dependencies supposed to be transitive in Gradle?